isis/config/rclone.conf

```diff
@@ -84,6 +84,10 @@ remote = naif:/pub/naif/pds/data/jno-j_e_ss-spice-6-v1.0/jnosp_1000/data/
 type = alias
 remote = asc_s3:asc-isisdata/usgs_data/juno
 
+[lo_usgs]
+type = alias 
+remote = asc_s3:asc-isisdata/usgs_data/lo 
```

Comment:
Seems like we don't really use the same kernels as what is on NAIF. So I only put USGS source.